### ISSUE TITLE
VS Code extension icon assets type and encoding

### ISSUE THREAD

**Issue by xqdoo00o** [microsoft/vscode#158435]
For now, vs code extension's icon use the image/png type and gzip encoding(like this [python extension icon](https://ms-python.gallerycdn.vsassets.io/extensions/ms-python/python/2022.12.1/1660785997548/Microsoft.VisualStudio.Services.Icons.Default)).
The gzip encoding is not useful for png format(and basically every bitmap format), icon size even bigger after gziped which only increase useless cpu load. 
And the png format is a little outdated, how about using webp format? or using svg format since most extension icon could be vector graphics?

**Comment by isidorn** [comment]
Correct. I think there are no plans to change this on the Marketplace side.

fyi @prashantvc
